packages/blockchain-link-types/src/blockbook-api.ts

```diff
@@ -46,6 +49,7 @@ export interface MultiTokenValue {
     value?: string;
 }
 export interface TokenTransfer {
+    /** @deprecated: Use standard instead. */
```

Comment:
nice, who is going to replace it everywhere? @tomasklim ?

Reply:
I think it should be done